programs/tests/c-tests/gpio/gpio.c

```diff
@@ -1,6 +1,16 @@
 #include <stdint.h>
 #include <flexpret.h>
 
+/**
```

Comment:
This is connected to the pin_event server. If I remember correctly, then the GPI_1 refers to 32 pins not a single one.